### PR TITLE
File Errors

### PR DESCRIPTION
**Pull request by noahsegal**

### CODE REVIEW

File: resources/client/write-test.txt [hunk 1]
```diff
@@ -28,3 +28,20 @@ Here comes the sun, and I say
 It's all right
 It's all right
 
+THIS FILE IS A SAMPLE FOR WRITING TO THE SERVER
```
Comment (irushavm) [inline resources/client/write-test.txt]: Appended text is not needed.

File: resources/config/config.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "DEBUG_MODE": false,
-  "SERVER_PORT": 69,
-  "SIMULATOR_PORT": 23,
+  "DEBUG_MODE": true,
+  "SERVER_PORT": 8069,
```
Comment (irushavm) [inline resources/config/config.json]: plz.

File: resources/server/read-test.txt [hunk 3]
```diff
@@ -1,30 +0,0 @@
-THIS FILE IS A SAMPLE FOR READING FROM THE SERVER
-
-Here comes the sun (doo doo doo doo)
-Here comes the sun, and I say
```
Comment (irushavm) [inline resources/server/read-test.txt]: Why is this deleted?

File: src/main/java/states/ReadState.java [hunk 4]
```diff
@@ -48,60 +52,129 @@ public ReadState(SocketAddress serverAddress, String filename, boolean isVerbose
...
-		int expBlockNum = -1;
-		try {
-			// Set up + Send RRQ Message
+
+        expBlockNum = -1;
+        isConsentual = true;
+        try {
+            // Create the resource manager, handle IOException if failed to get resource directory
+
+            socket = new TFTPDatagramSocket();
```
Comment (irushavm) [inline src/main/java/states/ReadState.java]: The socket is initialized in the constructor. This is done to run the mock integration tests properly

File: src/main/java/states/ReadState.java [hunk 5]
```diff
@@ -25,6 +24,11 @@
 	private SocketAddress serverAddress;
 	private String filename;
 
+    private static final Logger LOG = new Logger("FTPClient - Read");
+    private static final String READ_OPERATION = "read";
```
Comment (irushavm) [inline src/main/java/states/ReadState.java]: Is this used anywhere?

File: resources/client/write-test.txt (hunk 1 above)
Comment (noahsegal) [reply resources/client/write-test.txt]: Issue with the .gitignore not ignoring the resources/ folders. This is being worked on.

File: resources/server/read-test.txt (hunk 3 above)
Comment (noahsegal) [reply resources/server/read-test.txt]: Please see above

File: src/main/java/states/ReadState.java (hunk 4 above)
Comment (noahsegal) [reply src/main/java/states/ReadState.java]: That was a mistake on my part. Missed it during a rebase. Will fix.

File: src/main/java/states/ReadState.java (hunk 5 above)
Comment (noahsegal) [reply src/main/java/states/ReadState.java]: It is not...not sure why that appeared under my changes. Will delete.

File: resources/config/config.json (hunk 2 above)
Comment (noahsegal) [reply resources/config/config.json]: Issue with .gitignore

**Comment by JFleming4** [comment]
Make sure to rebase again with my changes
